src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -49,8 +48,22 @@ public CommandResult execute() {
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
+
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            Set<String> wordsInNameLowerCase = new HashSet<>();
+            Set<String> keywordsLowerCase = new HashSet<>();
+
+            Iterator<String> it = wordsInName.iterator();
+            while(it.hasNext()){
```

Comment:
Let's stick to a consistent style and keep the spacing for the `while` loops, i.e. `while (it.hasNext()) {`